### PR TITLE
Improve language

### PR DESCRIPTION
**Pull request by aleemont1**
Correzioni varie su forma e lingua.

### CODE REVIEW

**Comment by aleemont1** [comment]
Farò uno squash quando finisco di sistemare e se approvate i commit.

**Review by michelececcacci — APPROVED** [review]
LGTM, remember to squash and use english in PRs

File: 13-template.tex [hunk 1]
```diff
@@ -47,16 +47,19 @@ \subsubsection{Functional Requirements}
...
+	\item Should meet minimal security requirements for user recognition.
+	\item Should be smart enough to consistenly manage the whole house-system.  
 \end{itemize}
 
 \section{Analysis and domain model}
 
-The application will emulate a domotic controller connected to home devices.
-The home devices will also be emulated, each with its own logic.
-The user will be able to interact with the devices via the controller with a dashboard.
+The application will emulate a domotic environment, some smart devices and facilities monitoring emulation.
```
Comment (smagnaniuni) [inline 13-template.tex]: What do you mean with "and facilities monitoring emulation"? Do you mean "and facilities monitoring the emulation"? But I don't think that's the case in our model.

File: 13-template.tex [hunk 2]
```diff
@@ -31,8 +31,8 @@ \chapter{Analysis}
 
 \section{Requirements}
 
-The goal of the software HOMER (HOMe EmulatoR) is to emulate a domotic controller connected to lights, windows, and other domestic devices.
-The software will provide a dashboard to the user to allow control of the devices and monitoring of sensors and consumptions.
+HOMER (HOMe EmulatoR) is an emulated domotic controller connected to lights, windows, and other domestic facilities.
+The smart environment is mananged through a dashboard which allows the user to control devices, monitor sensors and consumptions.
```
Comment (smagnaniuni) [inline 13-template.tex]: Typo "mananged" should be corrected to "managed"

File: 13-template.tex [hunk 3]
```diff
@@ -47,16 +47,19 @@ \subsubsection{Functional Requirements}
...
-The application will emulate a domotic controller connected to home devices.
-The home devices will also be emulated, each with its own logic.
-The user will be able to interact with the devices via the controller with a dashboard.
+The application will emulate a domotic environment, some smart devices and facilities monitoring emulation.
+Smart devices are common domestic devices as lights, electrical outlets, windows, doors and various sensors 
+for monitoring indoor temperature and air quality along with electrical absorption.
+The user will interact with the devices via the controller through a dashboard.
 
-The main difficulty will be modelling the communication between the controller and the different devices, which can vary in functioning.
+The main difficulty will be implementing a communication protocol for the controller and the devices, which can vary in functioning and tecchnical aspects.
```
Comment (smagnaniuni) [inline 13-template.tex]: Typo "tecchnical" should be corrected to "technical"

File: 13-template.tex (hunk 1 above)
Comment (aleemont1) [reply 13-template.tex]: Facilities in this context means 'impianti' or 'servizi'

File: 13-template.tex (hunk 2 above)
Comment (aleemont1) [reply 13-template.tex]: will fix ASAP

File: 13-template.tex (hunk 3 above)
Comment (aleemont1) [reply 13-template.tex]: Will fix ASAP

Comment (michelececcacci) [inline 13-template.tex]:
```suggestion
The main difficulty will be implementing a communication protocol for the controller and the devices, which can vary in functioning and technical aspects. 
```

Comment (michelececcacci) [reply 13-template.tex]: @smagnaniuni next time you can just directly suggest the change like this. it's less error prone. UPDATE:
actually, you can just commit the change directly from the github UI

**Review by michelececcacci — CHANGES_REQUESTED** [review]
just the small change mentioned in the comments.

File: 13-template.tex (hunk 1 above)
Comment (smagnaniuni) [reply 13-template.tex]: Ok I think I understand what you mean ("e servizi/impianti di monitoraggio"), there are two problems:
1. saying emulation again is a repetition
2. the ~~adjective~~ participle "monitoring" should be before the noun

The final form should be "and monitoring facilities"

Comment (aleemont1) [reply 13-template.tex]: No, in this case "facilites" is the subject of the phrase "facilities emulation", but yeah, emulation is repeated, therfore should be deleted.

**Comment by aleemont1** [comment]
I think I fixed everything, I'll proceed to squash and push when you confirm there's no more errors.

**Review by michelececcacci — APPROVED** [review]
LGTM, remember to squash

**Pull request merged**